File: LeetCode/Medium/2482-difference-between-ones-and-zeros-in-row-and-column/2482-difference-between-ones-and-zeros-in-row-and-column.go
func onesMinusZeros(grid [][]int) [][]int {
    rowLen := len(grid)
    colLen := len(grid[0])
    oneRows := make([]int, rowLen)
    oneCols := make([]int, colLen)

    for row := 0; row < rowLen; row++ {
        temp := 0
        for col := 0; col < colLen; col++ {
            temp += grid[row][col]
        }

        oneRows[row] = temp * 2 - rowLen
    }

    for col := 0; col < colLen; col++ {
        temp := 0
        for row := 0; row < rowLen; row++ {
            temp += grid[row][col]
        }

        oneCols[col] = temp * 2 - colLen
    }

    for row := 0; row < rowLen; row++ {
        for col := 0; col < colLen; col++ {
            grid[row][col] = oneRows[row] + oneCols[col];
        }
    }

    return grid
}

File: LeetCode/Medium/2482-difference-between-ones-and-zeros-in-row-and-column/2482-difference-between-ones-and-zeros-in-row-and-column.cpp
class Solution {
public:
    vector<vector<int>> onesMinusZeros(vector<vector<int>>& grid) {
        int rowLen = grid.size();
        int colLen = grid[0].size();
        int oneRows[rowLen];
        int oneCols[colLen];

        for (int row = 0; row < rowLen; ++row){
            int temp = 0;
            for (int col = 0; col < colLen; ++col){
                temp += grid[row][col];
            }
            
            oneRows[row] = temp * 2 - rowLen;
        }

        for (int col = 0; col < colLen; ++col){
            int temp = 0;
            for (int row = 0; row < rowLen; ++row){
                temp += grid[row][col];
            }

            oneCols[col] = temp * 2 - colLen;
        }

        for (int row = 0; row < rowLen; ++row){
            for (int col = 0; col < colLen; ++col){
                grid[row][col] = oneRows[row] + oneCols[col];
            }
        }
        
        return grid;
    }
};